dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/UnpackagedTests/UnpackagedTests.csproj

```diff
@@ -63,6 +63,9 @@
     <None Include="resources.pri.standalone">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include=".runsettings">
```

Comment:
can we give it a name? Starting with a dot makes it a generated file.

Reply:
Sure, how about "`UnpackagedTests.runsettings`".